src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,101 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class CalculatorTest {
+    private Calculator calculator;
+
+    CalculatorTest() {
+        calculator = new Calculator();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"", " ", "   "})
```

Comment:
`@NullAndEmptySource`를 사용하면 어떨까요? 아래의 글을 참고하세요.
https://www.baeldung.com/parameterized-tests-junit-5

```suggestion
    @NullAndEmptySource
    @ValueSource(strings = {" ", "   "})
```